DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/CourseContent/Index.cshtml

```diff
@@ -34,9 +34,12 @@
       </div>
     </div>
 
-    @foreach (var section in Model.Sections)
-    {
-      <partial name="_CourseSectionExpandable" model="section" />
+    @if (Model.Sections.Any(s => s.Tutorials.Any(t => t.DiagnosticEnabled || t.LearningEnabled) || s.PostLearningAssessment)) {

```

Comment:
This is a bit difficult to read - can we pull this into a derived boolean on the model that can be unit tested?